### PR TITLE
Update packages + rename drone

### PR DESCRIPTION
**Pull request by lramos15**
Update the node packages
Lock docker to a specific version
Rename drone to deploy headsup

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 kind: pipeline
-name: deploy_chorebot
+name: deploy_headsup
```
Comment (ddbruce) [inline .drone.yml:2]: lol oops

File: src/crews.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ function buildDateArray() {
     const currentHour = crewDate.hours();
     // If it's not noon yet we use the previous day's crew
     if (currentHour < 12) {
-        crewDate.subtract(1, "days");
+        crewDate.subtract(1, 'days');
```
Comment (ddbruce) [inline src/crews.js:40]: Could've sworn our eslint did doubles

**Review by ddbruce — APPROVED** [review]

File: src/crews.js (hunk 2 above)
Comment (lramos15) [reply src/crews.js:40]: https://github.com/rpiambulance/headsup/blob/6010c3aad8beb179fde8f582026a33a40f8bf9ec/.eslintrc.json#L18

Comment (ddbruce) [reply src/crews.js:40]: No, I know. I checked. Just thought our default eslint did doubles

**Pull request merged**